specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/preview/2022-01-15-privatepreview/common.json

```diff
@@ -0,0 +1,208 @@
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {
+    "ProvisioningState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "The current provisioning state.",
+      "enum": [
+        "Succeeded",
+        "Updating",
+        "Canceled",
+        "Deleting",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "OperationalState": {
+      "type": "string",
+      "description": "Operational state for the resource.",
+      "enum": [
+        "Configuring",
+        "Succeeded",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "operationalState"
+      }
+    },
+    "AnnotationResource": {
+      "type": "object",
+      "description": "Switch configuration entries require a description to discern between configuration groups.",
+      "properties": {
+        "annotation": {
+          "type": "string",
+          "description": "Switch configuration description."
+        }
+      }
+    },
+    "ExtendedLocation": {
+      "type": "object",
+      "description": "The extended location.",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The extended location type."
+        },
+        "name": {
+          "type": "string",
+          "description": "The extended location name."
+        }
+      }
+    },
+    "ReachabilityState": {
+      "type": "string",
+      "description": "Generic network reachability state.",
+      "enum": [
+        "Reachable",
+        "Unreachable"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "reachabilityState"
+      }
+    },
+    "EnabledDisabledState": {
+      "type": "string",
+      "description": "EnabledDisabledState state for the resource.",
+      "readOnly": true,
+      "enum": [
+        "Enabled",
+        "Enabling",
+        "Disabling",
+        "Disabled"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "enabledDisabledState"
+      }
+    },
+    "FailedSucceededState": {
+      "type": "string",
+      "description": "FailedSucceeded state for the resource.",
+      "enum": [
+        "Succeeded",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "FailedSucceededState"
+      }
+    },
+    "UpdateAdministrativeState": {
+      "type": "object",
+      "description": "Update administrative state on list of resources.",
+      "properties": {
+        "state": {
+          "type": "string",
+          "description": "Administrative state.",
+          "enum": [
+            "Enable",
+            "Disable"
+          ],
+          "x-ms-enum": {
+            "name": "AdministrativeState",
+            "modelAsString": true
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/EnableDisableOnResources"
```

Comment:
The examples show these POST actions taking an l2IsolationDomains resource ID array.  However, its a POST action on a single l2ISolationDomains resource and shouldn't affect any other resources (RBAC, etc... would be a concern).  What is the intention of the resource ID array?